build.gradle.kts

```diff
@@ -19,8 +19,10 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
   implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:5.2.1")
 
-  implementation("com.amazonaws:aws-java-sdk-s3:$awsSdkVersion")
+  testImplementation("software.amazon.awssdk:s3:2.30.0")
   implementation("com.amazonaws:aws-java-sdk-sts:$awsSdkVersion")
```

Comment:
I don't think this is actually required

Reply:
I meant that I don't think `aws-java-sdk-sts` is required here, although interesting that we can do without S3 as well. 

https://mvnrepository.com/artifact/software.amazon.sns/sns-extended-client/2.1.0 is bringing in S3